resources/assets/js/components/Upcoming.js

```diff
@@ -19,15 +19,15 @@ export default class Upcoming extends Component {
   }
 
   renderColumns() {
-    const todayDate = new Date(this.state.today);
+    let todayDate = new Date(this.state.today);
     return (
       [...Array(7)].map((_, i) => {
-        const thisDay = new Date(todayDate);
+        let thisDay = new Date(todayDate);
         thisDay.setDate(todayDate.getDate() + i);
-        const thisEvents = this.state.events.filter(e => {
+        let thisEvents = this.state.events.filter(e => {
           return e.date == thisDay.getDate() && e.month == thisDay.getMonth() + 1
         });
-        return <UpcomingColumn key={"column"+i} start={thisDay} today={todayDate} events={thisEvents} />
+        return <UpcomingColumn key={"column"+i} i={i} start={thisDay} today={i == 0} tomorrow={i == 1} events={thisEvents} />
```

Comment:
No point passing through i, today, tomorrow. You can just pass through i and work out today and tomorrow in the UpcomingColumn widget

Reply:
yeah i would recommend having a boolean called isToday and that's it